src/components/project/ProjectBasicsPane.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import FormControl from "@material-ui/core/FormControl";
+import TextField from "@material-ui/core/TextField";
+import OrganizationSelect from "../../components/general/OrganizationSelect.js";
+
+export default function ProjectBasicsPane(props) {
+  return (
+    <>
+      <OrganizationSelect
+        organizations={props.organizations}
+        handleChange={props._handleOrganizationChange}
```

Comment:
It feels a little weird to pass a property prefixed with an underscore like this—that typically indicates that something is private. What's the reasoning for doing this?